### ISSUE TITLE
VNC lost connection about per 20 seconds

### ISSUE THREAD

**Issue by zeerd** [AndronixApp/AndronixOrigin#74]
I am using OnePlus8T with Android 11.
The connection of VNC will lost about each 15~20 seconds, when the termux keeping on showing on the screen of my phone.

When the VNC lost connection, I need to touch the screen of my phone to re-active the termux again, And the VNC could work for the next 15~20 seconds.

I am not sure of that it's an issue of termux or andronix or the android os it-self.
Just looking for some suggest.

Updated: I am using the PC Mode to access the VNC Client , but not from a PC via network. This might be the problem. I tried to connect the VNC from another PC, the VNC could keep the connection well.

**Comment by ultrahacx** [comment]
Its the OxygenOS which is killing the termux. Follow this link and disable the battery optimization from termux
https://dontkillmyapp.com/oneplus

**Comment by zeerd** [comment]
Thank you. I did some trying.

I disabled both the Deep optimization  and the Sleep standby optimization .
And, did : Turn off System settings > Battery > Battery optimization, switch to ‘All apps’ in top right menu > Termux > Don’t optimize
And locked the Termux in recent app.

But, the issue still there.

PS: I did not find the " Auto-Launch" from the setting.

**Comment by ultrahacx** [comment]
Please join us at our [Discord server ](https://chat.andronix.app)or on our [forum ](https://forum.andronix.app)for more detailed discussions. Github is not managed for support and community is on above mentioned platforms.

**Comment by zeerd** [comment]
Somebody from Termux thought this is an issue of the H2OS for OnePlus. I will try to contact them and see what happens.

**Comment by zeerd** [comment]
I did more tests on this, and find, I might be wrong.
It's not an issue of Termux or Andronix or standard Android OS. It might be an issue of PC Mode(Desktop Mode) of Android.

I tried to run two different VNC Clients. One on the home screen of Phone, One on the screen of PC Mode.
When the issue occur. The client on Home Screen is still working , well the client on PC Screen is freezing.

So , means, the VNC Server is good, but the application on PC Screen is freezing.
But I am still confuse that why only VNC Client has this issue. (If I use chrome on PC Mode, everything good).

I will try to commit an issue for this to android.
